ui/src/app/kvm/components/KVMResourcesCard/_index.scss

```diff
@@ -2,6 +2,16 @@
   $donut-size: 6.5rem;
   $donut-width: 14px;
 
+  .kvm-machine-list-modal {
+    max-width: none;
+    min-width: 0;
+    width: 35rem;
+
+    @media only screen and (max-width: $breakpoint-small) {
+      width: auto;
```

Comment:
I'm not sure `auto` width makes sense in the context of an absolutely positioned element like this dropdown. I think `vw` units would be a better fit. Getting the edges to align *perfectly* will take a bit of width calculation though. Regardless, the "auto" contextual menu should probably be implemented first.